src/lib/modules/Document/DocumentInfo.js

```diff
@@ -58,6 +70,10 @@ export class DocumentInfo extends Component {
               <Table.Cell width={4}>Title</Table.Cell>
               <Table.Cell>{metadata.title}</Table.Cell>
             </Table.Row>
+            <Table.Row>
+              <Table.Cell width={4}>Alternative titles</Table.Cell>
+              {this.renderAlternativeTitles(metadata.alternative_titles)}
```

Comment:
what will be displayed if there is no alternative titles?

In general I would like to avoid creating component methods and try to keep JSX in the render function when possible. Another way would be creating a functional component (which might already exist for displaying a list) ?

Reply:
just an empty list item, same as the rest in that section.